data.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Agumon', DATE '2020-02-03',0,true,10.23);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Gabumon', DATE '2018-11-15',2,true,8.00);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Pikachu', DATE '2021-01-07',1,false,15.04);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Devimon', DATE '2017-05-12',5,true,11.00);
```

Comment:
- [ ] You created `animals` table NOT `animal`. Below is the error message I got while trying to fill your table with these data. Kindly insert into the  `animals` table, not `animal` table because you don't have that table called `animal`. You only have `animals` table.
### Error
![query-error](https://github.com/amanuelbelete95/vet-clinic-database/assets/26597432/2795aeca-ad71-42f8-b72b-1d6aa48ecef5)